### DISCUSSION TITLE
openelb vip mode chose interface bond

### DISCUSSION THREAD

**Discussion by huabanyu8411** [openelb/openelb#397] [Q&A]
env :
openelb install in kubernetes cluster, every kubernetes node have three NICs, openelb use vip mode.
question:
The host has three NICs, how does OpenELB bind to one of them?

**Comment by renyunkang** [comment]
We have not yet supported this parameter, the nodes default interface will be used.

**Reply by renyunkang** [reply]
https://github.com/openelb/openelb/issues/398
